File: reactjs/src/services/ticket/dto/simpleTicketDto.ts
import { EntityDto } from "../../dto/entityDto";
import {TicketPriority} from './ticketPriority';
import {TicketType} from './ticketType';

export class SimpleTicketDto extends EntityDto { 
    title!: string; 
    priority!: TicketPriority;
    type!: TicketType;
}

File: aspnet-core/src/TicketTracker.Application/Tickets/Dto/SimpleTicketDto.cs
﻿using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TicketTracker.Activities.Dto;
using TicketTracker.Components.Dto;
using TicketTracker.Entities;
using TicketTracker.Statuses.Dto;
using TicketTracker.Works.Dto;

namespace TicketTracker.Tickets.Dto {
    [AutoMap(typeof(Ticket))]
    public class SimpleTicketDto : EntityDto<int> {
        [Required]
        public string Title { get; set; } 
        public TicketPriority Priority { get; set; }
        public TicketType Type { get; set; } 
    }
}
